pkg/graphdb/graphdb.go

```diff
@@ -378,12 +378,22 @@ func (db *Database) Purge(id string) (int, error) {
 		tx.Rollback()
 		return -1, err
 	}
-
 	changes, err := rows.RowsAffected()
 	if err != nil {
 		return -1, err
 	}
 
+	// Clear who's using this id as parent
+	refs, err := tx.Exec("DELETE FROM edge WHERE parent_id = ?;", id)
```

Comment:
😢 I think this should be fine, but had the graph db on my list to delve into. I personally always like the db to be in control for referential integrity, not the app :)